daemon.lua

```diff
@@ -20,7 +22,14 @@ function telegraf:push(message)
 	self:sendto(message, inet.localhost, 8094)
 end
 
-local inbox = mailbox.inbox(100 * 1024)
+local inbox = {
+	telegraf = mailbox.inbox(100 * 1024),
+	reply    = mailbox.inbox(1024)
```

Comment:
why that small?

Reply:
I just thought the occurences would be seldom. Will stick to the same amount then.